src/components/PizzasLister.js

```diff
@@ -37,50 +48,91 @@ class PizzasLister extends Component {
                 copiedTempState.ingredients = response.data
                 copiedTempState.ready = true
                 this.setState(copiedTempState)
+                //It might not be necessary
+                this.props.saveIngredientsData(this.state)
             })
             .catch(error => { console.log(error) })
 
     }
 
+    pizzasListAppender(pizzasList){
+        
+        const emptyPizza = {
+            ingredients: [],
+            name: "Your Choice Pizza",
+            imageUrl: "https://upload.wikimedia.org/wikipedia/commons/thumb/0/06/20140704-0532_Balboa_Peninsula.JPG/320px-20140704-0532_Balboa_Peninsula.JPG"
+        }        
+        
+        pizzasList.unshift(emptyPizza)
+    }
+
     ingredientsLister(pizzaIngredientsIds) {
         return this.state.ingredients.filter(
             element => pizzaIngredientsIds.includes(element.id)
         )
     }
 
-    ingredientsStringCreator(pizzaIngredientsIds){
-       return this.ingredientsLister(pizzaIngredientsIds)
-       .map(ingredient => {return ingredient.name})
-       .join(", ")
+    ingredientsStringCreator(pizzaIngredientsIds) {
+        return this.ingredientsLister(pizzaIngredientsIds)
+            .map(ingredient => { return ingredient.name })
+            .join(", ")
     }
 
-    pizzaPriceCalculator(pizzaIngredientsIds){
-       return this.ingredientsLister(pizzaIngredientsIds)
-       .map(ingredient => {return ingredient.price})
-       .reduce((acc, current)=>acc+current, this.state.basePrice)
+    pizzaPriceCalculator(pizzaIngredientsIds) {
```

Comment:
Rather name methods as verbs or action likeL calculatePizzaPrice. Nouns rather refer to objects and classes. If i saw just the name of this method without any context, i'd think that it's a class.